addons/mail/static/src/core/web/activity_menu.js

```diff
@@ -57,7 +57,7 @@ export class ActivityMenu extends Component {
         ];
     }
 
-    openActivityGroup(group) {
+    openActivityGroup(group, filter="all") {
```

Comment:
Side note: could you fix commit message (cfr guidelines) ? Like [FIX] mail: ... Could you also link tasks / tickets using standard notation like "Task-3890068 OPW-3787054" ?

Otherwise LGTM :)

Reply:
Thank you for the suggestion. I have made the suggested changes.